nuxeo-web-ui-ftest/test/pages/ui/selection.js

```diff
@@ -8,4 +8,12 @@ export default class Selection extends BasePage {
     this.el.element('nuxeo-clipboard-documents-button').click();
     this.waitForNotVisible();
   }
+
+  moveDown() {
+    this.el.element('nuxeo-move-documents-down-button').click();
+  }
+
+  moveUp() {
+    this.el.element('nuxeo-move-documents-up-button').click();
```

Comment:
Same as above, would insert a `this.el.waitForVisible('nuxeo-move-documents-up-button')` before this line.